src/cpp/ext/otel/otel_plugin.cc

```diff
@@ -264,4 +264,76 @@ void OpenTelemetryPluginBuilder::BuildAndRegisterGlobal() {
 }
 
 }  // namespace internal
+
+namespace experimental {
+absl::string_view OpenTelemetryClientAttemptStartedInstrumentName() {
```

Comment:
We may have already discussed this, but is there a reason why these need to be functions instead of just constexpr constants?

Reply:
I had tried that some years ago but had ran into issues. Let me try it again and see if it works now.

Reply:
pushed a canary change to see if the tests are happy with it. If they are, I'll use it for all.

Reply:
seems to be working. Replaced all